# HG changeset patch
# User David Bateman <dbateman@free.fr>
# Date 1239742995 -7200
#      Tue Apr 14 23:03:15 2009 +0200
# Node ID 7d903c71470edc813addd6ed50a57c9bd447013e
# Parent  3b810beddfa64d947c1fe399b17f325d70e0eac1
Keep __marching_cube__ undocumented for now

diff --git a/scripts/plot/__marching_cube__.m b/scripts/plot/__marching_cube__.m
--- a/scripts/plot/__marching_cube__.m
+++ b/scripts/plot/__marching_cube__.m
@@ -11,16 +11,22 @@
 ## GNU General Public License for more details.
 ##
 ## You should have received a copy of the GNU General Public License
 ## along with this program; if not, see http://www.gnu.org/licenses/gpl.html.
 
 ## -*- texinfo -*-
 ## @deftypefn  {Function File} {[@var{t}, @var{p}] =} __marching_cube__ (@var{x}, @var{y}, @var{z}, @var{val}, @var{iso})
 ## @deftypefn  {Function File} {[@var{t}, @var{p}, @var{c}] =} __marching_cube__ (@var{x}, @var{y}, @var{z}, @var{val}, @var{iso}, @var{col})
+## Undocumented internal function.
+## @end deftyefn
+
+## -*- texinfo -*-
+## @deftypefn  {Function File} {[@var{t}, @var{p}] =} __marching_cube__ (@var{x}, @var{y}, @var{z}, @var{val}, @var{iso})
+## @deftypefn  {Function File} {[@var{t}, @var{p}, @var{c}] =} __marching_cube__ (@var{x}, @var{y}, @var{z}, @var{val}, @var{iso}, @var{col})
 ##
 ## Return the triangulation information @var{t} at points @var{p} for
 ## the isosurface values resp. the volume data @var{val} and the iso
 ## level @var{iso}.  It is considered that the volume data @var{val} is
 ## given at the points @var{x}, @var{y} and @var{z} which are of type
 ## three--dimensional numeric arrays.  The orientation of the triangles
 ## is choosen such that the normals point from the higher values to the
 ## lower values.
